app.js

```diff
@@ -0,0 +1,160 @@
+// HTML elements
+const statusDiv = document.querySelector('.status');
+const resetDiv = document.querySelector('.reset');
+const cellDivs = document.querySelectorAll('.game-cell');
+let player1 = 'player 1';
+let player2 = 'player 2';
+const startBtn = document.getElementById('start');
+const newGame = document.getElementById('game');
+const gamePlayers = document.getElementById('players');
+const clearBtn = document.getElementById('clear');
+
+// Game variables
+
+let gameStart = true;
+let currentPlayer = true;
+
+
+// functions
+
+const handleWin = (player) => {
+    gameStart = false;
+    if (player === 'x') {
+        statusDiv.innerHTML = `${player1} has won`;
+    } else {
+        statusDiv.innerHTML = `<span>${player2} has won</span>`;
+    }
+}
+
+const checkGameStatus = () => {
```

Comment:
This particular function would be a lot simpler if the player variables are made to be a factory function. 

You need to store the player moves then use the current move of each player to decide if that player is a winner or not.
![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/objects_requirement.png)